packages/ra-tree-ui-materialui/src/DeleteWithConfirmMenuItem.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { Fragment, Component, ReactElement } from 'react';
```

Comment:
This should be in ra-ui-material-ui. Same with other menu items.

Reply:
Do we need them elsewhere ?